src/packages/core/tests/server.test.ts

```diff
@@ -190,10 +192,12 @@ describe("server", () => {
     });
 
     it("listens on given interface only", async function () {
-      // skip this test unless in CI, as this test iterates over all available network interfaces
-      // and network interfaces on user machines are unpredictible and may behave in ways that
-      // we don't care about.
-      if (process.env.CI) this.skip();
+      // This test iterates over all available network interfaces. This can be problematic on user
+      // machines (which may be configured in unsupported ways), and also in macOS (which exposes
+      // unsupported interfaces). In these cases, we skip this test.
+      if (process.env.CI !== "true" || process.platform === "darwin") {
+        this.skip();
+      }
```

Comment:
I'm keen to hear your thoughts on this, but it appears to be problematic only on macOS, so I've allowed it to run on linux / windows in CI only.

Reply:
👍